apps/a11y-tests/src/tests/__snapshots__/ComponentExamples.test.tsx.snap

```diff
@@ -1250,7 +1250,49 @@ Array [
 
 exports[`a11y test checks accessibility of DetailsList (DetailsList.CustomFooter.Example) 1`] = `Array []`;
 
-exports[`a11y test checks accessibility of DetailsList (DetailsList.CustomGroupHeaders.Example) 1`] = `Array []`;
+exports[`a11y test checks accessibility of DetailsList (DetailsList.CustomGroupHeaders.Example) 1`] = `
+Array [
+  Object {
+    "kind": "fail",
+    "level": "error",
+    "locations": Array [
+      Object {
+        "logicalLocations": Array [
+          Object {
+            "fullyQualifiedName": ".cellIsGroupExpander-8",
+            "kind": "element",
+          },
+        ],
+        "physicalLocation": Object {
+          "artifactLocation": Object {
+            "index": 0,
+            "uri": "about:blank",
+          },
+          "region": Object {
+            "snippet": Object {
+              "text": "<div class=\\"ms-DetailsHeader-cell cellIsGroupExpander-8\\" data-is-focusable=\\"true\\" aria-expanded=\\"true\\" role=\\"button\\"><i data-icon-name=\\"ChevronRightMed\\" role=\\"presentation\\" aria-hidden=\\"true\\" class=\\"ms-DetailsHeader-collapseButton collapseButton-33\\"></i></div>",
+            },
+          },
+        },
+      },
+    ],
+    "message": Object {
+      "markdown": "Fix any of the following:
```

Comment:
I think I got it figured it out. These new accessibility errors are popping out now because you added the `role=“button”` to a div element and now the CustomGroupHeader and Grouped.Large examples need to have the new prop added to them. Before when it was just a regular div the tests were passing without having an aria-label on them but apparently the role button gives it a different meaning that requires it to have.

Reply:
Yes, with role="button" aria-label needs added to the snapshot. How can @mashiau update that?

Reply:
It’s like I mentioned in my comment above:

> … and now the CustomGroupHeader and Grouped.Large examples need to have the new prop added to them

Reply:
@mashiau you need to add the aria label to all the examples that are rendering this button.